### PR TITLE
Set-up pkgdown site and vignette to execute workflow

### LINKED ISSUE #56: Make vignette
Make `initialize.R` into a vignette.

### PR DESCRIPTION
**Pull request by kelshmo**
This PR fixes #56 and sets up the `pkgdown` architecture to host a homepage, function references and vignettes on a website. 

- The "start-up" script to initialize the workflow is removed and replaced by a vignette that delineates the steps in code chunks in this PR.

- I removed the bit about Docker for now, since I need to invest some time to update that resource.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,12 @@
 <!-- badges: start -->
   [![R build status](https://github.com/kelshmo/sageseqr/workflows/R-CMD-check/badge.svg)](https://github.com/kelshmo/sageseqr/actions)
 <!-- badges: end -->
-# sage-seqr
-RNASeq normalization pipeline with R.
+# RNA-seq normalization workflow in R
 
-This repository provides R markdowns for performing covariate adjustments and differential expression analysis of the RNA-seq data from AMP-AD and a Docker environment, thus all package dependencies are loaded. 
+The `sageseqr` package integrates the [`drake` R package](https://github.com/ropensci/drake/), the [`config` package for R](https://cran.r-project.org/web/packages/config/vignettes/introduction.html) and [Synapse](https://www.synapse.org/). `drake` tracks dependency relationships in the workflow and only updates data when it has changed. A `config` file allows inputs and parameters to be explicitly defined in one location. Synapse is a data repository that allows sensitive data to be [stored and shared responsibly](https://docs.synapse.org/articles/article_index.html#governance).
```
Comment (jaybee84) [inline README.md:6]: The `sageseqr` package integrates the [`drake` R package](https://github.com/ropensci/drake/), the [`config` package for R](https://cran.r-project.org/web/packages/config/vignettes/introduction.html), and [Synapse](https://www.synapse.org/).

Comment (jaybee84) [reply README.md:6]: added a missing comma :)

File: README.md [hunk 2]
```diff
@@ -1,68 +1,12 @@
...
-password: <password>
-  
-4. You can `docker stop <container name>` to pause your work and return to it later with `docker start <container name>`.
-
-* To find your container name `docker ps -a`
-
-# Clone the Repo
-
-`git clone https://github.com/kelshmo/sageseqr.git` to get the `sageseqr` tool in your Docker repository.
+The workflow takes RNA-seq gene counts and sample metadata as inputs, normalizes counts by CQN, removes outliers based on a user-defined threshold, empirically selects meaningful covariates and returns differential expression analysis results.
```
Comment (jaybee84) [inline README.md:8]: Maybe add citation for CQN

File: vignettes/run-the-plan.Rmd [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+3. Load the `sageseqr` library and login to [Synapse](https://www.synapse.org/). `rnaseq_plan()` calls the arguments from the config file and creates the `drake` plan. Execute `drake::make(plan)` to compute.
+
+```{r run-plan}
+library(sageseqr)
+
+# Login to Synapse. Make a Synapse account and use synapser to login: https://r-docs.synapse.org/articles/manageSynapseCredentials.html
+synapser::synLogin()
+
+# Run the analysis
+plan <- sageseqr::rnaseq_plan(metadata_id = config::get("metadata")$synID,
```
Comment (jaybee84) [inline vignettes/run-the-plan.Rmd:35]: is it safe to assume that rnaseq_plan has been defined elsewhere? My knowledge of drake is super limited but the example I have noted generally use "drake_plan"

File: vignettes/run-the-plan.Rmd [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                    continuous_input = config::get("continuous"),
+                    gene_id = config::get("biomart")$`gene id`,
+                    biomart_id = config::get("biomart")$synID,
+                    biomart_version = config::get("biomart")$version,
+                    filters = config::get("biomart")$filters,
+                    host = config::get("biomart")$host,
+                    organism = config::get("biomart")$organism
+)
+
+drake::make(plan)
```
Comment (jaybee84) [inline vignettes/run-the-plan.Rmd:50]: wonder if there is a use case where "clean()" should be used before make(), to clean other author's results.

File: vignettes/run-the-plan.Rmd (hunk 3 above)
Comment (kelshmo) [reply vignettes/run-the-plan.Rmd:35]: It is since I wrote `rnaseq_plan` to wrap `drake_plan`!

File: README.md (hunk 2 above)
Comment (kelshmo) [reply README.md:8]: Good point - I'm not sure I have seen software cited in a README. For now I will add a hyperlink, but look into example projects.

File: vignettes/run-the-plan.Rmd (hunk 4 above)
Comment (kelshmo) [reply vignettes/run-the-plan.Rmd:50]: If the user was installing this package or cloning it from the repo, they would not have any results locally. I will add an issue to document when and how to use `clean()`. Thanks!

**Pull request merged**